internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -184,6 +185,122 @@ func (ctx *VMContext) Send(to address.Address, method types.MethodID, value type
 	return out, ret, nil
 }
 
+// Send allows actors to invoke methods on other actors
+func (ctx *VMContext) Send(to address.Address, method types.MethodID, value types.AttoFIL, params []interface{}) interface{} {
```

Comment:
This method is mostly the same as `LegacySend`. You should extract that code into a new function rather than copying it. This will preserve git history when `LegacySend` goes away.